data/json/npcs/NC_TRUE_FOODPERSON.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "type": "item_group",
+    "id": "NC_TRUE_FOODPERSON_worn",
+    "subtype": "collection",
+    "entries": [
+      { "item": "foodperson_mask" },
+      { "item": "briefs" },
+      { "item": "socks" },
+      { "item": "pants_cargo" },
+      { "item": "dress_shirt" },
+      { "item": "chestguard_hard" },
+      { "item": "cape_fp" },
+      { "item": "armguard_hard" },
+      { "item": "legguard_hard" },
+      { "item": "gloves_rubber" },
+      { "item": "boots_rubber" }
+    ]
+  },
+  {
+    "id": "NC_TRUE_FOODPERSON_melee",
+    "type": "item_group",
+    "items": [ [ "bat", 50 ], [ "null", 50 ] ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_TRUE_FOODPERSON_misc",
+    "subtype": "collection",
+    "entries": [
+      { "item": "medium_disposable_cell", "prob": 90 },
+      { "item_group": "snacks_fancy", "prob": 30, "charges": 2 },
```

Comment:
You should use `group`, not `item_group` here and on the line below.

Reply:
Done #31730

Reply:
Thanks